AWS_FLB_CHERRY_PICKS

```diff
@@ -23,3 +23,16 @@ https://github.com/PettitWesley/fluent-bit.git s3-log-key-warn-1_9 308b73558fb2d
 
 # ECS Filter
 https://github.com/PettitWesley/fluent-bit.git ecs-empty-metadata-fix-one-commit 24934c938e8f63700edf1230a09e4483ac5df6a3
+
+# Add back Datadog fixes that do not trigger segfault
+https://github.com/matthewfala/fluent-bit.git ecs-datadog-sequential-revert acc01a4bf4ce656023d82943bec2683b5b7755e5
+https://github.com/matthewfala/fluent-bit.git ecs-datadog-sequential-revert ac30b7c876a95d44c33a406b2b58ce08b978587e
+
+# Datadog Partial Fix Patch
+https://github.com/matthewfala/fluent-bit.git datadog-ecs-patch 3c1ad69ada5bb6f2e448c6f39a1a0ea6a6f4ff17
+
+# S3 retry limit configuration option
+https://github.com/Claych/fluent-bit.git clay-retry-limit-1.9 26922abb9b970c322d658117df68228dc26df5f9
+https://github.com/Claych/fluent-bit.git clay-retry-limit-1.9 a5410e4d3bda21f5072f55c8e6a128654a9b9894
+https://github.com/Claych/fluent-bit.git clay-retry-limit-1.9 f0fe0245228330a9200325dcc2d594d4e3d49e89
+https://github.com/Claych/fluent-bit.git clay-retry-limit-1.9 96587eebdb91d1f4d37c705e54e1ae91327f3353
```

Comment:
does this need to be a bunch of commits? I think the PR could be squashed to 1.

Reply:
Squashed. Thanks Clay